.github/workflows/ci.yml

```diff
@@ -29,12 +48,31 @@ jobs:
           toolchain: stable
           components: clippy
 
-      - run: cargo clippy --workspace --all-features -- -D warnings
+      - run: cargo clippy -p ${{ matrix.crate }} --no-deps --all-features -- -D warnings
 
   rustfmt:
     if: ${{ github.ref == 'refs/heads/master'
             || startsWith(github.ref, 'refs/tags/deadpool-')
             || !contains(github.event.head_commit.message, '[skip ci]') }}
```

Comment:
@elpiel additionally, would love to see if you strip this condition from CI pipeline, since GitHub now supports `[skip ci]` out-of-the-box, so they're just redundant.